src/strings/PowerFxResources.en-US.resx

```diff
@@ -5139,7 +5139,7 @@
     <comment>Error Message.</comment>
   </data>
   <data name="WrnDelagationTableNotSupported" xml:space="preserve">
```

Comment:
>WrnDelagationTableNotSupported [](http://example.com/codeflow?start=14&length=30)

Where is this used? If we are changing the string from expecting one argument to expecting two, we need to pass the additional argument to it.

Reply:
This is going to be used from Dataverse project side. Please check PR https://github.com/microsoft/Power-Fx-Dataverse/pull/187.

Reply:
We shouldn't have a resource from this project that is not used here. This should be moved to the PowerFx-Dataverse project instead.